### DISCUSSION TITLE
管理 GitOps 中的 Secret：Argocd-Vault-Plugin | CloudNative.love 💕

### DISCUSSION THREAD

**Discussion by giscus** [SJFCS/cloudnative.love#8] [Announcements]
# 管理 GitOps 中的 Secret：Argocd-Vault-Plugin | CloudNative.love 💕

Argocd-Vault-Plugin(AVP) 是一个自定义的 ArgoCD 插件，这个插件旨在帮助解决 GitOps 和 Argo CD 的 Secret 管理问题。用于从 HashiCorp Vault 等秘密管理器中检索秘密并将其注入 Kubernetes YAML 文件中，从而无需依赖 Operator 或 Custom Resource，这使得它不仅可以用于 Secrets 资源，还可以能够 parameterize 任何 Kubernetes 资源，甚至是自定义资源。

https://www.cloudnative.love/blog/2022/07/23/%E7%AE%A1%E7%90%86%20GitOps%20%E4%B8%AD%E7%9A%84%20Secret%EF%BC%9AArgocd-Vault-Plugin/README

**Comment by nishuzumi** [comment]
git@github.com:SJFCS/arogcd-vault-plugin-demo.git 已经不存在了
